ietf/templates/doc/ad_list.html

```diff
@@ -87,6 +103,37 @@ <h2 class="mt-5" id="{{ dt.type.0 }}">{{ dt.type.1 }} State Counts</h2>
     </script>
     <script src="{% static "ietf/js/highcharts.js" %}"></script>
     {{ data|json_script:"data" }}
+
+    <script>
+         Array.from(document.querySelectorAll("table"))
+            .filter(table => {
+                const hasDataSum = Boolean(table.querySelector("[data-sum]"))
+                return hasDataSum
```

Comment:
Nitpicking here: avoid creating a variable that's never used afterward. return the result directly.

Reply:
Fixed